modules/services/swayidle.nix

```diff
@@ -114,7 +123,7 @@ in {
           "${cfg.package}/bin/swayidle -w ${concatStringsSep " " args}";
       };
 
-      Install = { WantedBy = [ "sway-session.target" ]; };
+      Install = { WantedBy = [ cfg.systemdTarget ]; };
```

Comment:
Hmm, if swayidle works with any compositor, wouldn't `graphical-session.target` be the most appropriate value? Seems unnecessary to have to set it to whichever compositor you happen to use.

Reply:
> Hmm, if swayidle works with any compositor, wouldn't `graphical-session.target` be the most appropriate value? Seems unnecessary to have to set it to whichever compositor you happen to use.

The way it was merged allows swayidle to be linked to another target than "graphical-session.target" being 
 more flexible.
The Default option could be "graphical-session.target", following the implemented in kanshi, wlsunset and waybar.